pyiceberg/table/snapshots.py

```diff
@@ -19,15 +19,40 @@
     Any,
     Dict,
     List,
+    Mapping,
     Optional,
 )
 
 from pydantic import Field, PrivateAttr, model_serializer
 
 from pyiceberg.io import FileIO
-from pyiceberg.manifest import ManifestFile, read_manifest_list
+from pyiceberg.manifest import DataFile, DataFileContent, ManifestFile, read_manifest_list
 from pyiceberg.typedef import IcebergBaseModel
 
+ADDED_DATA_FILES = 'added-data-files'
+ADDED_DELETE_FILES = 'added-delete-files'
+ADDED_EQUALITY_DELETES = 'added-equality-deletes'
+ADDED_FILE_SIZE = 'added-files-size'
+ADDED_POSITION_DELETES = 'added-position-deletes'
+ADDED_POSITION_DELETE_FILES = f'{ADDED_POSITION_DELETES}-files'
```

Comment:
This isn't correct. It should be `added-position-delete-files` but will be `added-position-deletes-files` instead. Same with the other strings constructed with references.